src/components/book.js

```diff
@@ -0,0 +1,27 @@
+const Book = () => (
+  <>
+    <div className="card">
+      <div className="main-area">
+        <span>Action</span>
+        <h1>The hunger games</h1>
+        <p>Unseen Content</p>
+        <ul>
+          <li>Corn</li>
+          <li>Corn</li>
+          <li>Corn</li>
+        </ul>
+      </div>
+
+      <div className="main-area2">
+        <h4>60%</h4>
+      </div>
+      <div className="main-area3">
+        <h4>Current Chapter</h4>
+        <h5>Chapter 16</h5>
+        <button type="button">UPDATE CHAPTER</button>
+      </div>
+    </div>
+  </>
+);
+
+export default Book;
```

Comment:
- You have done well so far, but according to the project requirement, your `Book` component takes the book data as a prop and renders the title and the author, please use this [article](https://legacy.reactjs.org/docs/components-and-props.html) for a better understanding 🙏 
- Your book component should also contain a button for removing books (don't implement this functionality yet), the button only serves as a placeholder 😄 . Though styling is not required reference this design for [visual](https://pedantic-beaver-cb923b.netlify.app/#) purposes only 👍